### DISCUSSION TITLE
FDA Project Kanban

### DISCUSSION THREAD

**Discussion by HenryGeorgist** [HydrologicEngineeringCenter/HEC-FDA#101] [Q&A] [answered]
Our current FDA Project Kanban is specific to the HEC-FDA repository, that means it is not possible (that I can find) to add tickets from the fda-model repository.

If we create a FDA Project Kanban at the HEC organization, we can add tickets across multiple repositories. Would that be something we should invest time in?

@rnugent3 it is up to you.

**Comment by rnugent3** [comment, marked as answer]
Yes, please. It would be useful for us to track everything in one place.
Thank you.

**Reply by HenryGeorgist** [reply]
Sorry, I somehow missed this until now. Making an org level Kanban now.

**Reply by HenryGeorgist** [reply]
https://github.com/orgs/HydrologicEngineeringCenter/projects/5

Done, added Someday Maybe, and Backlog.  You can change the Kanban of a particular issue on the issue itself - so it is not a huge deal to move tickets over on a per ticket basis, but it might take a moment or two to move it for all open tickets on the HEC-FDA repository.
